lib/msal-browser/FAQ.md

```diff
@@ -59,15 +59,34 @@
 
 ## What browsers are supported by MSAL.js?
 
-MSAL.js has been tested with the following browsers:
+MSAL.js has been tested and supports the last 2 stable and supported versions of the following browsers:
 
-IE 11, Edge, Chrome, Firefox and Safari
+- Chrome
+- Edge (Chromium)
+- Firefox
+- Safari
+- Opera
 
-Keep these steps in mind when [using MSAL.js with IE](https://github.com/AzureAD/microsoft-authentication-library-for-js/wiki/Using-msal.js-with-Internet-Explorer).
+MSAL.js has also been tested and supports the following browsers with Promise polyfills:
```

Comment:
Isn't this contrasting the statement above: "Despite this, IE 11 is currently supported but we do not include Promise polyfills"

Reply:
Never mind, is this 1.0?

Reply:
This is true for both v1 and v2. We do not include the polyfills, developers need to provide them and this is described in more detail in the IE doc.

Reply:
The statement is confusing in that case. Can we always explicitly mention that we do not include polyfills?

Reply:
Updated.